runtime/parachains/src/paras_inherent.rs

```diff
@@ -1164,12 +1159,13 @@ mod tests {
 				assert_eq!(<scheduler::Pallet<Test>>::scheduled(), vec![]);
 
 				assert_eq!(Pallet::<Test>::on_chain_votes(), None);
-				// Call enter with our 2 backed candidates
+				// Call enter with our 2 disputes
 				assert_ok!(Pallet::<Test>::enter(
 					frame_system::RawOrigin::None.into(),
 					limit_inherent_data,
 				));
 
+				// Ensure that calling enter with 3 disputes would cause an overlength block
```

Comment:
This is not correct - this error is because `enter` is already called for this block. If you want to check that `Ensure that calling enter with 3 disputes would cause an overlength block` it would be easisest to setup the scenario again in another integration test

To get around just this error you could clear the `Included` storage item , but then we will have other issues because the previous call to `enter` already modified storage, hence why I think the easiest/best way to test this is by creating a new scenario

Reply:
`overlength` -> `over weight`